pkgs/applications/emulators/nuked-md/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, gitUpdater
+, cmake
+, SDL2
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nuked-md";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "nukeykt";
+    repo = "Nuked-MD";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-Pe+TSu9FBUhxtACq+6jMbrUxiwKLOJgQbEcmUrcrjMs=";
+  };
+
+  postPatch = ''
+    # Interesting detail about our SDL2 packaging:
```

Comment:
This huge comment should be outside the phase, because
1. It is interpreted by the shell
2. Because of the previous point, any modification to this comment generates a no-op rebuild

Reply:
I don't really expect to change the comment until the used `SDL2` doesn't exhibit this problem, at which point the part this is a comment to will be obsolete & removed anyway, but I generally agree with you on this. I'm just more used to reviewers complaining about the exact opposite situation, demanding to move the comment into the phase…

Reply:
This is one of those problematic quirks of interfacing with other things besides Nix.